### ISSUE TITLE
Rekit plugin loader seems to break after installing other libraries

### ISSUE THREAD

**Issue by loganpowell** [supnate/rekit#34]
I've done this multiple times at this point, so I think I've narrowed the problem down, but it seems that after `rekit install redux-saga` rekit portal is loading. But, once I try to install reactstrap (a UI library) it busts with this: 
```
Warning: Failed to load plugin: redux-saga, Error: Cannot find module 'C:/Users/surface/Projects/Bootcamp/groups/rekit-01/node_modules/rekit-plugin-redux-saga/config'.
Error: Cannot find module 'C:/Users/surface/Projects/Bootcamp/groups/rekit-01/node_modules/rekit-plugin-redux-saga/config'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at plugins.map (C:\Users\surface\Projects\Bootcamp\groups\rekit-01\node_modules\rekit-core\core\plugin.js:77:22)
    at Array.map (native)
    at Object.loadPlugins (C:\Users\surface\Projects\Bootcamp\groups\rekit-01\node_modules\rekit-core\core\plugin.js:75:21)
    at Object.<anonymous> (C:\Users\surface\Projects\Bootcamp\groups\rekit-01\node_modules\rekit-portal\middleware\index.js:19:18)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
```

**Comment by loganpowell** [comment]
when I try to create an async action it gives me `TypeError: Cannot read property 'hooks' of null`

**Comment by supnate** [comment]
Have you upgraded rekit-portal to 2.0.4? It should have fixed the problem.

**Comment by supnate** [comment]
Hi @loganpowell , the first issue has been fixed in rekit-portal 2.0.5. Thanks very much for reporting this. If there's still problems for you, just reopen the issue.
